src/app/api/v1/user/welcome-scan/create/route.ts

```diff
@@ -57,7 +57,10 @@ export async function POST(
     getCountryCode(headers()),
   );
   if (!eligible) {
-    throw new Error("User is not eligible for feature");
+    logger.error("scan_created_error", {
```

Comment:
Do you think it's an error or just a warning? Anyone can try to do this if they hit the endpoint with a non-US IP.